emr/application/views/insights/main.php

```diff
@@ -64,25 +63,49 @@
+                              <td><?php echo $availability['first_name'];?></td>
+                              <td><?php echo $availability['last_name'];?></td>
+                              <td><?php echo $availability['employeeid'];?></td>
+                              <td><?php echo date("M jS, Y", strtotime($availability['timestamp'])); ?></td>
+                              <td><?php echo date("M jS, Y", strtotime($availability['start']));?> to <?php echo date("M jS, Y", strtotime($availability['start']));?></td>
+                              <td><?php echo $availability['reason'];?></td>
+                              <td><a class="btn btn-danger btn-xs" style="background-color: #32CD32; border-color: #32CD32" readonly> Approved </a></td>
+                            </tr>
+                          <?php } elseif($availability['approved']==FALSE && $availability['approved']!=NULL) { ?>
+                            <tr style="background-color: rgba(128, 0, 0, 0.3); color: black;">
+                              <td><?php echo $availability['first_name'];?></td>
+                              <td><?php echo $availability['last_name'];?></td>
+                              <td><?php echo $availability['employeeid'];?></td>
+                              <td><?php echo date("M jS, Y", strtotime($availability['timestamp'])); ?></td>
+                              <td><?php echo date("M jS, Y", strtotime($availability['start']));?> to <?php echo date("M jS, Y", strtotime($availability['start']));?></td>
+                              <td><?php echo $availability['reason'];?></td>
+                              <td><a class="btn btn-danger btn-xs" style="background-color: #800000; border-color: #800000;" readonly> Declined </a></td>
+                            </tr>
+                          <?php } else { ?>
+                            <tr style="background-color: none;">
+                              <td><?php echo $availability['first_name'];?></td>
+                              <td><?php echo $availability['last_name'];?></td>
+                              <td><?php echo $availability['employeeid'];?></td>
+                              <td><?php echo date("M jS, Y", strtotime($availability['timestamp'])); ?></td>
+                              <td><?php echo date("M jS, Y", strtotime($availability['start']));?> to <?php echo date("M jS, Y", strtotime($availability['start']));?></td>
+                              <td><?php echo $availability['reason'];?></td>
                               <td>
-                                <a data-toggle="modal" class="btn btn-danger btn-xs" data-target="#deleteClient"><i class="fa fa-trash-o"></i> Decline </a>
-                                <a href="" class="btn btn-success btn-xs"><i class="fa fa-pencil"></i> Accept </a>
+                                <?php echo form_open('request/updateRequest'); ?>
```

Comment:
why do you open up an `updateRequest` form here? are we allowing screeners to make changes to their submitted requests?